cves/kernel/CVE-2019-19052.yml

```diff
@@ -266,21 +116,21 @@ sandbox:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
-ipc:
-  question: |
-    Did the feature that this vulnerability affected use inter-process
-    communication? IPC includes OS signals, pipes, stdin/stdout, message
-    passing, and clipboard. Writing to files that another program in this
-    software system reads is another form of IPC.
-
-    Answer must be true or false.
-    Write a note about how you came to the conclusions you did, regardless of
-    what your answer was.
-  answer:
-  note:
+upvotes: "1"
+CWE_note: |
+  CWE as registered in the NVD. If you are curating, check that this
+  is correct and replace this comment with "Manually confirmed".
+mistakes: It appears when the device driver was written, there was a presumption
+  that deinitialization would occur smoothly. The classic "make sure to click
+  eject". This vulnerability is quite old, likely back when many Linux
+  developers held to this logic firmly. It is important to ensure that
+  developers account for funky ways of killing their programs without even
+  logging into the system.
```

Comment:
For multiline YAML, you should have a | at the end the mistakes line and then start your text on a newline